app/src/main/java/org/dhis2/usescases/datasets/datasetInitial/DataSetInitialActivity.java

```diff
@@ -165,7 +166,7 @@ public void showCatComboSelector(String catOptionUid, List<CategoryOption> data)
             if (selectedCatOptions == null)
                 selectedCatOptions = new HashMap<>();
             selectedCatOptions.put(catOptionUid, data.get(0));
-        } else {
+        } else if (data.size() <= 15) {
```

Comment:
15?¿

Reply:
It is the default limit we have for this kind of search dialog. I'll extract it to a constant